NEWS.md

```diff
@@ -63,35 +78,65 @@
   * Timeouts when connecting to a network server were fixed (thanks
     @bradc6).
   * A long-standing bug where some visplane overflows caused crashes was
-    fixed (thanks Mike Francis).
+    fixed (thanks Michael Francis).
   * A multiplayer deadlock bug where clients would stop sending tics after
     missing tics from the server was fixed. There are both client- and
     server- side fixes to fix the problem when playing with older versions
     (thanks MadDog and Mortrixs for help tracking this down).
+  * The macOS launcher now quits automatically when all windows are closed.
+  * The mouse speed calibration thermometer was fixed.
+  * Some improvements have been made to PC speaker emulation to better match Vanilla (thanks NY00123).
+  * Filenames and paths with non-latin characters now work on Windows (thanks Roman Fomin).
 
 ### Doom
+  * Doom 1.2 demo support was added (thanks James Canete!)
   * Map33 intermission screen and map33-map35 automap names are
     emulated (thanks CapnClever).
-  * We now exit gracefully when player starts are missing (thanks Mike
+  * We now exit gracefully when player starts are missing (thanks Michael
     Francis).
   * We now exit gracefully on levels with a boss brain and no boss spitter
     things (thanks Jason Benaim).
   * It's now possible to play multiplayer with gameversion=1.2.
+  * When the game hits the limit for the number of wall scrollers, a count
+    of the number of scrollers is now displayed (thanks Jon).
+  * Sector special 17 (random light flicker) is now accurately emulated to only appear in gameversion > 1.2 (thanks tpoppins).
+  * Fixed mouse events causing unwanted key presses (thanks Michael Day).
+  * French Doom II files doom2f.wad and french.deh are now automatically detected (thanks Acts19quiz).
+  * Mouse buttons are now bindable to run (thanks Archenoth).
```

Comment:
It is now even possible to bind mouse buttons to turn left/right in all games:
https://github.com/chocolate-doom/chocolate-doom/commit/331aed42b4490388d0d60867a731b7b7e63e8f3e